dialect/sql/schema/postgres.go

```diff
@@ -526,3 +526,94 @@ func arrayType(t string) bool {
 	}
 	return true
 }
+
+func (d *Postgres) foreignKeys(ctx context.Context, tx dialect.Tx, tables []*Table) (map[*Table][]*ForeignKey, error) {
+	var fks = make(map[*Table][]*ForeignKey)
+
+	var tableLookup = make(map[string]*Table)
+
+	// TODO: include schema
+	for _, t := range tables {
```

Comment:
I feel like I am not properly accounting for tables in different schemas but I don't believe this is current supported anyway